### DISCUSSION TITLE
考虑支持下快手小程序呗(#^.^#)

### DISCUSSION THREAD

**Discussion by darkfiredarkhalo** [NervJS/taro#11053] [Q & A]
[快手小程序](https://mp.kuaishou.com/docs/develop/guide/introduction.html)和微信,字节等的小程序类似, 考虑下支持呗, 不然还需要单独实现一遍(#^.^#)

**Comment by Chen-jj** [comment]
https://github.com/NervJS/taro-plugin-platform-kwai

**Comment by darkfiredarkhalo** [comment]
刚看到,已经支持了, 太棒了, https://github.com/NervJS/taro-plugin-platform-kwai
